charts/worker/values.yaml

```diff
@@ -58,4 +58,6 @@ bootConfig:
   config:
     managementPort: 35142
     backend: "kubernetes"
+    mongodbUrl: mongodb://root:secret@mongodb.skyramp:27017
```

Comment:
can we use something different to username and password?

Reply:
default credentials updated to admin/pa$$w0rd